### DISCUSSION TITLE
How to deloy agent on workstation

### DISCUSSION THREAD

**Discussion by xSunShine** [Security-Onion-Solutions/securityonion#9757] [Unsupported Versions]
Hello,

i know this seems stupid to ask but i installed SO in different types and they all worked but when it comes to agents i go to dowloads then pick a agent(which should be already configured with my ip adress and everything throw installation proces?) i download it and install it on the workstation which i test was linux and i dont get any logs from or any information that agent connected to manager panel? im new to SO so please give some advice. Also in distrbuted mode i configured everything but surricata wasnt generating any logs, so i switch to standalone which i left couple of hourse ago to see if surricata will do any sniffing on my network.

**Comment by cm-ops** [comment]
Would you be able to provide some more information? See https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 What agent are you trying to install and configure?

**Reply by xSunShine** [reply]
I am curious to ask does from download section from SO where i go and download agents and just install them on workstation or after download i have to configure them on workstation after installation. :) we will go deeper into this,just as if u can answer me this.

**Reply by cm-ops** [reply]
If you would provide some more information, at a minimum, what type of agent are you trying to install on the endpoints?

**Reply by xSunShine** [reply]
for example if i go to download section and choose wazuuh or windows agent, and i  install that agent on machine, do i have to do some kind of configuration at that machine or agent which downloaded from installed SO collects my ip and alll the neccessery things? and i guess in alerts i would see alerts for that kinda, btw i solved distrbuted installation with sensor

**Reply by cm-ops** [reply]
There are some configurations once downloaded, for example Winlogbeat - https://docs.securityonion.net/en/latest/beats.html#winlogbeat. Also, Wazuh - https://docs.securityonion.net/en/latest/wazuh.html#adding-agents.

**Reply by xSunShine** [reply]
i will read the docs u provided but i managed to install wazuh and to create auth key from cmd and it connected to manager node. thx for the docs
